.github/workflows/open-issue-in-repo.yml

```diff
@@ -0,0 +1,136 @@
+# **what?**
+# Open an issue in a specified repository with a specified title and body.  If the issue already exists, do nothing.
+# The calling workflow should be triggered off a PR.
+
+# **why?**
+# To be able to trigger issues to open in other repositories from a workflow in a different repository.
+
+# **when?**
+# This will be triggered from another workflow on a PR.  This could be triggered when a label is added to a PR.
+
+
+name: Open Issue in Another Repository
+
+on:
+  workflow_call:
+    inputs:
+      issue_repository:
+        description: 'Repository to open the issue in'
+        required: true
+        type: string
+      issue_labels:
+        description: 'Labels to add to the issue'
+        required: false
+        type: string
+      issue_title:
+        description: 'Title of the issue'
+        required: true
+        type: string
+      issue_body:
+        description: 'The body of the issue'
+        required: true
+        type: string
+    secrets:
+      FISHTOWN_BOT_PAT:
+        description: 'PAT to use for authentication'
+        required: true
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  issues: write # opens new issues
+  pull-requests: write # comments on PRs
+
+jobs:
+  prep:
+    runs-on: ubuntu-latest
+    outputs:
+      exists: ${{ steps.comment_check.outputs.exists }}
+
+    steps:
+
+      - name: "[DEBUG] Print variables"
+        id: print_variables
+        run: |
+          echo "Repository name to open the issue:      ${{ inputs.issue_repository }}"
```

Comment:
Since this is a debug step, I feel like we should use the variable names (e.g. issue_repository), and we should display all inputs (issue_labels).

Reply:
Done